backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/ClusterBuilder.java

```diff
@@ -122,19 +131,18 @@ public Cluster build() {
 
         username.map(username ->
             password.map(password ->
-                clusterBuilder.withCredentials(username, password)
-            )
-        );
+                clusterBuilder.withCredentials(username, password)));
```

Comment:
Maybe I am a bit picky but those diff should be squash in the commit that introduce those line